### PR TITLE
Update _payment.md

### PR DESCRIPTION
**Pull request by KBbitsP**

### CODE REVIEW

File: source/includes/_payment.md [hunk 1]
```diff
@@ -2090,9 +1326,7 @@ class Payment {
    "auditLogs":[]
 }
 ```
```
Comment (reshmabidikar) [inline source/includes/_payment.md:1328]: Same comment as above, I think an empty body is sent, could you please confirm and update if required?

File: source/includes/_payment.md [hunk 2]
```diff
@@ -2312,9 +1461,6 @@ class Payment {
    "auditLogs":[]
```
Comment (reshmabidikar) [inline source/includes/_payment.md:1461]: Same comment as above, I think an empty body is sent, could you please confirm and update if required?

File: source/includes/_payment.md [hunk 3]
```diff
@@ -2515,9 +1596,6 @@ class Payment {
...
```
Comment (reshmabidikar) [inline source/includes/_payment.md:1596]: Same comment as above, I think an empty body is sent, could you please confirm and update if required?

File: source/includes/_payment.md [hunk 4]
```diff
@@ -2769,9 +1724,6 @@ class Payment {
...
```
Comment (reshmabidikar) [inline source/includes/_payment.md:1724]: Same comment as above, I think an empty body is sent, could you please confirm and update if required?

File: source/includes/_payment.md [hunk 5]
```diff
@@ -3124,9 +2003,6 @@ class Payment {
...
 }
 ```
```
Comment (reshmabidikar) [inline source/includes/_payment.md:2005]: Same comment as above, I think an empty body is sent, could you please confirm and update if required?

File: source/includes/_payment.md [hunk 6]
```diff
@@ -3239,41 +2115,16 @@ paymentApi.create_payment_custom_fields(payment_id,
...
-    name: Test Custom Field
-    value: test_value
-    auditLogs: []
+```json
+{
+  "customFieldId":"7fb3dde7-0911-4477-99e3-69d142509bb9",
+  "objectId":"4927c1a2-3959-4f71-98e7-ce3ba19c92ac",
+  "objectType":"PAYMENT",
+  "name":"Test Custom Field",
+  "value":"test_value",
```
Comment (reshmabidikar) [inline source/includes/_payment.md:2124]: Same comment as above, I think an empty body is sent, could you please confirm and update if required?

File: source/includes/_payment.md [hunk 7]
```diff
@@ -3643,41 +2429,17 @@ paymentApi.create_payment_tags(payment_id,
...
-    tagDefinitionName: foo
-    auditLogs: []
+```json
+{
+  "tagId":"a46cfeb6-e175-42db-be62-7f117326ab4e",
+  "objectType":"PAYMENT",
+  "objectId":"28af3cb9-275b-4ac4-a55d-a0536e479069",
+  "tagDefinitionId":"353752dd-9041-4450-b782-a8bb03a923c8",
+  "tagDefinitionName":"foo",
+  "auditLogs":[]
```
Comment (reshmabidikar) [inline source/includes/_payment.md:2439]: Same comment as above, I think an empty body is sent, could you please confirm and update if required?

File: source/includes/_payment.md [hunk 8]
```diff
@@ -3742,64 +2504,16 @@ paymentApi.get_payment_tags(payment_id, api_key, api_secret)
...
-<
-[
-  {
+```json
+{
     "tagId": "890e3b13-3114-478b-9365-50f1a2682143",
     "objectType": "PAYMENT",
     "objectId": "8fe697d4-2c25-482c-aa45-f6cd5a48186d",
     "tagDefinitionId": "353752dd-9041-4450-b782-a8bb03a923c8",
     "tagDefinitionName": "foo",
```
Comment (reshmabidikar) [inline source/includes/_payment.md:2513]: Same comment as above, I think an empty body is sent, could you please confirm and update if required?

File: source/includes/_payment.md [hunk 9]
```diff
@@ -3945,12 +2646,8 @@ account.audit_logs_with_history(options)
...
 
-```shell
-# Subset of headers returned when specifying -v curl option
-< HTTP/1.1 200 OK
-<
-[
-  {
+```json
+{
     "changeType": "INSERT",
```
Comment (reshmabidikar) [inline source/includes/_payment.md:2651]: Is the response correct, the response should be an array, I think you missed adding `[`

File: source/includes/_payment.md [hunk 10]
```diff
@@ -4109,11 +2720,7 @@ AuditLogs logs = paymentApi.getPaymentAttemptAuditLogsWithHistory(paymentAttempt
...
 > Example Response:
 
-```shell
-# Subset of headers returned when specifying -v curl option
-< HTTP/1.1 200 OK
-< Content-Type: application/json
-[
+```json
   {
     "changeType": "INSERT",
```
Comment (reshmabidikar) [inline source/includes/_payment.md:2725]: The response should be an array, I think you missed adding [

**Review by reshmabidikar — COMMENTED** [review]
Looks good. One major issue that I found was that in many places, the API endpoint returns an empty response with a location header, however in the `Example Response`, you've specified the JSON that is obtained after querying the location header. Could you please make the changes for this as discussed on Slack?

File: source/includes/_payment.md (hunk 9 above)
Comment (KBbitsP) [reply source/includes/_payment.md:2054]: Hi Reshma, fixed all the places where array was required.

File: source/includes/_payment.md (hunk 10 above)
Comment (KBbitsP) [reply source/includes/_payment.md:2130]: Hi Reshma, fixed all the places where array was required.

**Review by reshmabidikar — APPROVED** [review]
Looks good!

**Pull request merged**